pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -33,16 +34,24 @@ import (
 
 func TestApplyIndexerCluster(t *testing.T) {
 	funcCalls := []spltest.MockFuncCall{
-		{MetaName: "*v1.Secret-test-splunk-secrets"},
-		{MetaName: "*v1.Secret-test-splunk-stack1-indexer-secrets"},
+		{MetaName: fmt.Sprintf("*v1.Secret-test-%s", GetNamespaceScopedSecretName("test"))},
```

Comment:
You may want to be more explicit with these names, to also ensure that the methods `GetNamespaceScopedSecretName` and `GetVersionedSecretName` are returning the expected values

Reply:
Done. Changed it back to test GetNamespaceScopedSecretName and GetVersionedSecretName APIs